Controller/ConnectController.php

```diff
@@ -183,6 +183,10 @@ public function connectServiceAction(Request $request, $service)
             $accessToken = $session->get('_hwi_oauth.connect_confirmation.'.$key);
         }
 
+        if (!is_array($accessToken)) {
```

Comment:
I'm not an expert on this bundle, when I debug the script, I saw $accessToken is an array, for facebook it is holding the $accessToken['access_token'] variable, but I don't know for other services the access_token key may change, and this is unified controller, I suppose asking is_array will solve the conflict which crash with following function.
public function getUserInformation(array $accessToken, array $extraParameters = array())

as you see the first parameter is array type safe.

But sure asking if(is_array()) just saves the fatal error.

Btw, I gave more details in this issue #796